src/hooks/useParams/useParams.ts

```diff
@@ -0,0 +1,20 @@
+import { useParams as ReactUseParams } from 'react-router-dom';
+
+import { PathParams } from 'types/params-utils';
+import { Simplify } from 'types/simplify';
+
+function assertParamsAreValid<Path extends string>(
```

Comment:
could you convert it to lambda?

Reply:
I think functions which are assertions have to be named function definitions 🤔 not sure if I am right

Reply:
you're right ☝️